### ISSUE TITLE
"Limit detectable fonts" appears to not work

### ISSUE THREAD

**Issue by AceSlash** [dillbyrne/random-agent-spoofer#450]
Hello, I am testing my browser on https://panopticlick.eff.org and this add-on doesn't protect me from the "System Fonts" detection.

With the addon, I see a unique combination of system fonts, in fact that is the only parameter that is unique between my browser and the 129446 others.

I guess it's a bug since "Limit detectable fonts" is one of the listed feature. I guess the best thing to do is to advertise only the fonts "Wingdings 2, Wingdings 3 (via javascript)" like the Tor Browser does.

**Comment by B00ze64** [comment]
Works fine for me on FF38 (tested on Panopticlick) - in fact, as soon as I enable this option the fonts for GitHub changes to Arial or something equally ugly. The only fonts showing according to Panopticlick and those Wingdings 2/3 that you mention...

**Comment by AceSlash** [comment]
Ok after some tests, it was a clash with the add-on "Stop Fingerprinting". It's this one that changes the detected system fonts. Too bad because I used it to hide the "Browser Plugin Details", which random-agent-spoofer does not.

Now tihout it, I'm unique with my "Browser Plugin Details" :/ I guess there is no perfect solution hide plugins and system fonts at the same time and be anonymous...

Closing since it's not a bug.
